packages/connect-web/src/module/index.ts

```diff
@@ -80,6 +80,8 @@ const TrezorConnect = factory(
 export default TrezorConnect;
 export * from '@trezor/connect/src/exports';
 
-window.addEventListener('beforeunload', () => {
-    impl.dispose();
-});
+if (typeof window !== 'undefined') {
+    window.addEventListener('beforeunload', () => {
```

Comment:
this was also pretty werid though, `windows` hasn't been defined here. The code isn't neede for dev to run properly and the change here won't change much, but still. We should know abou tit.